### PR TITLE
update BaseFileHelper Completed

### PR DESCRIPTION
**Pull request by jackSpanrrows**
Signed-off-by: jackSpanrrows <15811072719@163.com>
BaseFileHelper 翻译完成
| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: framework/helpers/BaseFileHelper.php [hunk 1]
```diff
@@ -231,10 +231,10 @@ protected static function loadMimeTypes($magicFile)
     private static $_mimeAliases = [];
 
     /**
-     * Loads MIME aliases from the specified file.
-     * @param string $aliasesFile the path (or alias) of the file that contains MIME type aliases.
-     * If this is not set, the file specified by [[mimeAliasesFile]] will be used.
-     * @return array the mapping from file extensions to MIME types
+     * 从指定文件加载 MIME 别名。
+     * @param string $aliasesFile 包含 MIME 类型别名的文件的路径（或别名）。
+     * 如果没有设置，将使用 [[mimeMagicFile]] 指定的文件。
```
Comment (cuileon) [inline framework/helpers/BaseFileHelper.php]: 这里的变量名为什么写错了？

File: framework/helpers/BaseFileHelper.php [hunk 2]
```diff
@@ -251,47 +251,47 @@ protected static function loadMimeAliases($aliasesFile)
...
+     * @param string $src 源目录
+     * @param string $dst 目标目录
+     * @param array $options 目录复制选项。有效的选项是：
      *
-     * - dirMode: integer, the permission to be set for newly copied directories. Defaults to 0775.
-     * - fileMode:  integer, the permission to be set for newly copied files. Defaults to the current environment setting.
-     * - filter: callback, a PHP callback that is called for each directory or file.
-     *   The signature of the callback should be: `function ($path)`, where `$path` refers the full path to be filtered.
-     *   The callback can return one of the following values:
+     * - dirMode：整型，为新复制的目录设置的权限。默认为0775。
```
Comment (cuileon) [inline framework/helpers/BaseFileHelper.php]: 数字和汉字之间加空格

File: framework/helpers/BaseFileHelper.php [hunk 3]
```diff
@@ -730,18 +730,18 @@ private static function matchPathname($path, $basePath, $pattern, $firstWildcard
     }
 
     /**
-     * Scan the given exclude list in reverse to see whether pathname
-     * should be ignored.  The first match (i.e. the last on the list), if
-     * any, determines the fate.  Returns the element which
-     * matched, or null for undecided.
+     * 扫描非排除列表以便查看路径名是否被忽略。
+     * 第一个匹配（i.e. 列表内的最后一个），如果有任何可能性，
```
Comment (cuileon) [inline framework/helpers/BaseFileHelper.php]: i.e. 翻译为例如

**Comment by jackSpanrrows** [comment]
已提交一版修改后的

**Pull request merged**